src/pickers/_datepicker.scss

```diff
@@ -0,0 +1,343 @@
+        color: $datepicker-date-hover-color;
+        cursor: pointer;
+        outline: none;
+      }
+    }
+  }
+
+  &__previous-month {
+    float: left;
+  }
+
+  &__current-month {
+    display: inline-block;
+    font-weight: $datepicker-month-title-font-weight;
+    line-height: 32px;
+  }
+
+  &__next-month {
+    float: right;
+  }
+
+  &__week-days {
+    .mdl-datepicker__date {
+      color: $datepicker-week-days-color;
+    }
+  }
+
+  &__week {
+    clear: both;
+    margin: 2px 0;
+    overflow: hidden;
+  }
+
+  &__date {
+    // background-color: $datepicker-date-bg-color;
+    background-color: transparent;
+    border: none;
+    border-radius: 50%;
+    display: inline-block;
+    float: left;
+    font-size: $datepicker-date-font-size;
+    height: 36px;
+    line-height: 34px;
+    outline: none;
+    text-align: center;
+    vertical-align: middle;
+    width: 36px;
+    transition: background-color 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;
+
+    &--today {
+      color: $datepicker-today-color;
+      font-weight: $datepicker-today-font-weight;
+    }
+
+    &--empty {
+      cursor: default !important;
+      background-color: transparent !important;
+
+      &:hover {
+        cursor: default !important;
+        background-color: transparent !important;
+      }
+    }
+
+    &--disabled {
+      color: $datepicker-date-disabled-color;
+      cursor: default !important;
+      background-color: transparent !important;
+
+      &:active,
+      &:focus,
+      &:hover {
+        color: $datepicker-date-disabled-color !important;
+        cursor: default !important;
+        background-color: transparent !important;
+      }
+    }
+  }
+
+  &__year-picker {
+    display: none;
+    height: 235px;
+    overflow: auto;
+    position: relative;
+  }
+
+  &--year-picker.is-visible {
+    .mdl-datepicker__week-days,
+    .mdl-datepicker__navigation,
+    .mdl-datepicker__month {
+      display: none;
+    }
+
+    .mdl-datepicker__header {
+      &-year {
+        color: $datepicker-header-active-color;
+      }
+
+      &-date {
+        color: $datepicker-header-color;
+      }
+    }
+
+    .mdl-datepicker__year-picker {
+      display: block;
+    }
+  }
+
+  &__year {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    display: block;
+    font-size: $datepicker-font-size;
+    height: 32px;
+    line-height: 32px;
+    outline: none;
+    position: relative;
+    text-align: center;
+    width: 100%;
+
+    &--selected {
+      color: $datepicker-year-selected-color;
+      font-size: $datepicker-year-selected-font-size;
+      height: 40px;
+      line-height: 40px;
+    }
+
+    &--disabled {
+      &,
+      &:active,
+      &:focus {
+        cursor: default;
+        color: $datepicker-year-disabled-color;
+      }
+    }
+  }
+
+  &__actions {
+    margin: 0 -14px;
+    padding: $datepicker-actions-padding;
+    text-align: right;
+  }
+
+  // Dark theme
```

Comment:
Hmm, I'm not so sure if the build process is the right place; a user might need both dark and light elements simultaneously, since the choice merely depends on the background a widget is being placed on. They might need dark components for the parts of their page they've customised to be darker, or simply when placing them on the header instead of the content, for example.

Still, I agree that it may be best to leave this out for now, until we come up with an agreed-upon solution.